includes/domain-models/class-sensei-domain-models-model-abstract.php

```diff
@@ -361,6 +437,8 @@ protected static function meta_field() {
 	 * @return Sensei_Domain_Models_Field_Declaration_Builder Field declaration builder instance.
 	 */
 	protected static function derived_field() {
```

Comment:
This and the following methods doesn't have the `* @deprecated 3.11.0` comment.

Reply:
Fixed in ea178d50be79c54c7048ebd0680816f1068f1d44